### DISCUSSION TITLE
Splitter Audio problems, Home Assistant Integration and RPi Zero 2 W GPIO

### DISCUSSION THREAD

**Discussion by luppercut** [awawa-dev/HyperHDR#928] [Q&A]
Thanks for supporting this project. It's a lot of fun to tinker with.

For the most part I have it running with a few problems. I would appreciate any guidance on these points:

My EZCOO HDMI splitter causes audio drop out. I have higher quality cables hipping in to test that.
I've seen it mentioned that Home Assistant lights can be added, which would be awesome but I can't figure out hot to do that. Perhaps I need to upgrade to a beta version?
Currently I run HyperHDR on a Pi that's on the same TV as an SK6812 strip connected to an ESP that it's wirelessly connected to. In a previous iteration(older version of HyperHDR I think), I was able to control the strip directly from the Pi IO, but now I can't get that to work.

**Comment by awawa-dev** [comment]
Hi
I dont now what splitter model you have but maybe it is damaged or edid is set incorrectly? I cant help with Home Assistant because I don't use it and the integration with HA is an external project - maybe some other user could help here?
Need full logs taken 2-3 minutes after start to see what's going on with the LED driver. Try to play an effect then.

**Reply by luppercut** [reply]
Thanks for the reply!
Splitter model:
https://a.co/d/b3f0v79

I'm mainly unsure how to set it up with version 20. I chose GPIO piblaster and tried to set it to pin 4, which is what I think the data wire is connected to.
![Screenshot 2024-08-23 190238](https://github.com/user-attachments/assets/cbf20443-4178-4ab6-8af9-8f71cddddabe)
